CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Changed
 - Updated getting started to user guide ([#233](https://github.com/opensearch-project/opensearch-py/pull/233))
 - Updated CA certificate handling to check OpenSSL environment variables before defaulting to certifi ([#196](https://github.com/opensearch-project/opensearch-py/pull/196))
+- Modified index definition in client/__init__py([#245](https://github.com/opensearch-project/opensearch-py/pull/245))
```

Comment:
No need for a CHANGELOG entry for a doc change, you can remove. It needs an extra space before `(` otherwise I would have merged anyway :)